lms/static/sass/elements/_banners.scss

```diff
@@ -61,19 +61,31 @@ $full-width-banner-margin: 20px;
     }
 
     .alert {
-        margin-bottom: $baseline !important;
+        display: flex;
         padding: $baseline;
         border: 1px solid;
 
+        &:not(:last-child) {
+            margin-bottom: $baseline / 2;
+        }
+
         .icon-alert {
-            margin-right: $baseline / 4;
+            @include margin-right($baseline / 2);
+        }
+
+        .message-actions {
+            @include margin-left($baseline / 2);
         }
 
         &.alert-info {
```

Comment:
Hmm... NIT as well, but line-height: initial? The text lines are a little tight here on the message.